src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -1,85 +1,82 @@
 import React, { useCallback, useMemo, useState } from "react";
-import { Button, Nav } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import { Nav } from "react-bootstrap";
 import classNames from "classnames";
 
-import AdvancedSettings from "pages/Admin/AdvancedSettings";
-import BasicInfo from "pages/Admin/BasicInfo";
-import EntranceExperience from "pages/Admin/EntranceExperience";
-import VenueDetails from "pages/Admin/Venue/Details";
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useRoles } from "hooks/useRoles";
+import { useIsAdminUser } from "hooks/roles";
 
-import { Venue_v2 } from "types/venues";
+import { LoadingPage } from "components/molecules/LoadingPage";
 
 import "./AdminVenueView.scss";
 
 export enum AdminVenueTab {
-  dashboard = "dashboard",
-  basicInfo = "basic_info",
-  entranceExperience = "entrance_experience",
-  advancedMapSettings = "advanced_map_settings",
+  spaces = "spaces",
+  timing = "timing",
+  run = "run",
 }
 
-const adminVenueTabs: Record<AdminVenueTab, String> = {
-  [AdminVenueTab.basicInfo]: "Start",
-  [AdminVenueTab.entranceExperience]: "Entrance",
-  [AdminVenueTab.advancedMapSettings]: "Advanced",
-  [AdminVenueTab.dashboard]: "Dashboard",
+const adminVenueTabLabelMap: Readonly<Record<AdminVenueTab, String>> = {
+  [AdminVenueTab.spaces]: "Spaces",
+  [AdminVenueTab.timing]: "Timing",
+  [AdminVenueTab.run]: "Run",
 };
 
-const DEFAULT_TAB = AdminVenueTab.dashboard;
+const DEFAULT_TAB = AdminVenueTab.spaces;
 
-export interface AdminVenueViewProps {
-  venue: Venue_v2;
-}
+export const AdminVenueView: React.FC = () => {
+  const venueId = useVenueId();
+  const [selectedTab, setSelectedTab] = useState<AdminVenueTab>(DEFAULT_TAB);
 
-export const AdminVenueView: React.FC<AdminVenueViewProps> = ({ venue }) => {
-  const [selectedTab, setSelectedTab] = useState<string>(DEFAULT_TAB);
+  const { userId } = useUser();
+  const { roles } = useRoles();
+  const { isAdminUser } = useIsAdminUser(userId);
 
-  const selectDefaultTab = useCallback(() => {
-    setSelectedTab(DEFAULT_TAB);
-  }, []);
+  // Get and pass venue to child components when working on tabs
+  const { isCurrentVenueLoaded } = useConnectCurrentVenueNG(venueId);
 
   const renderAdminVenueTabs = useMemo(() => {
-    return Object.entries(adminVenueTabs).map(([id, text]) => (
+    return Object.entries(adminVenueTabLabelMap).map(([key, text]) => (
       <Nav.Link
-        key={id}
+        key={key}
         className={classNames("AdminVenueView__tab", {
-          "AdminVenueView__tab--selected": selectedTab === id,
+          "AdminVenueView__tab--selected": selectedTab === key,
         })}
-        eventKey={id}
+        eventKey={key}
       >
         {text}
       </Nav.Link>
     ));
   }, [selectedTab]);
 
+  const selectTab = useCallback((tab: string) => {
```

Comment:
```suggestion
  const selectTab = useCallback((tab: AdminVenueTab) => {
```

Reply:
Or as per https://github.com/sparkletown/sparkle/pull/1511#discussion_r649088680, just remove this and use `setSelectedTab`

Reply:
This value is NavLink's onSelect first parameter, it's already typed as string, I have to cast it otherwise I get compile error from TS.